domain-management/src/main/java/org/jboss/as/domain/management/parsing/AuditLogXml_4.java

```diff
@@ -639,175 +633,4 @@ private void parseAuditLogHandlersReference(final XMLExtendedStreamReader reader
             }
         }
     }
-
-    @Override
-    public void writeAuditLog(XMLExtendedStreamWriter writer, ModelNode auditLog) throws XMLStreamException {
```

Comment:
If StandaloneXml_5 or HostXml_5 have a reference to AuditLogXml_4 or AuditLogXml_Legacy, that's a bug and a failure is good. They are constructed with a ref to a Namespace that when passed to AuditLogXml.getInstance(ns) will not result in these impls being returned.